frontend/src/utils/TokenService.js

```
@@ -312,6 +313,49 @@ const TokenService = {
+  /**
+   * Get the last profile update timestamp
+   * @param {string} uid - The user's ID
+   * @returns {Date|null} The timestamp of the last update or null if not found
+   */
+  getProfileUpdateTime(uid) {
+    if (!uid) return null;
+    
+    const timestamp = localStorage.getItem(`${this.profileUpdateTimeKey}_${uid}`);
+    if (!timestamp) return null;
+    
+    return new Date(parseInt(timestamp));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)